test_aodncore/pipeline/test_storage.py

```diff
@@ -225,7 +225,7 @@ def test_set_is_overwrite_with_no_action_file(self):
 
         temp_file = PipelineFile(self.temp_nc_file)
         self.assertIsNone(temp_file.dest_path)
-        self.assertIs(temp_file.publish_type, PipelineFilePublishType.NO_ACTION)
+        self.assertIs(temp_file.publish_type, PipelineFilePublishType.UNSET)
```

Comment:
Update test method name too?
Is this the same test as `test_set_is_overwrite_with_unset_file` in pipeline/steps/test_store.py ?

Reply:
This one tests directly against a broker object, the other tests setting through the runner object. Looking at it, it's largely the same, but I think this was more of an internal test, whereas the other is the interface that the handler itself uses.